source/Applications/Basic/Visualization/CaptureHDRVisNormals/CaptureHDRVisNormals.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+This example shows how to capture Zivid point clouds, with color and normals.
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <pcl/visualization/cloud_viewer.h>
+
+#include <iostream>
+#include <thread>
+
+using namespace std::chrono_literals;
+
+namespace
+{
+    pcl::visualization::PCLVisualizer::Ptr normalsVisualizer(pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr cloud,
+                                                             pcl::PointCloud<pcl::PointXYZRGBNormal>::ConstPtr normals)
+    {
+        // --------------------------------------------------------
+        // -----Open 3D viewer and add point cloud and normals-----
+        // --------------------------------------------------------
+        auto viewer = boost::make_shared<pcl::visualization::PCLVisualizer>("Viewer");
+
+        int view_rgb(0);
+        viewer->createViewPort(0.0, 0.0, 0.5, 1.0, view_rgb);
+        viewer->addText("Cloud RGB", 0, 0, "RGBText", view_rgb);
+        viewer->addPointCloud<pcl::PointXYZRGB>(cloud, "cloud", view_rgb);
+
+        int view_normals(0);
+        viewer->createViewPort(0.5, 0.0, 1.0, 1.0, view_normals);
+        viewer->addText("Cloud Normals", 0, 0, "NormalsText", view_normals);
+        viewer->addPointCloud<pcl::PointXYZRGBNormal>(normals, "cloudNormals", view_normals);
+        viewer->addPointCloudNormals<pcl::PointXYZRGBNormal, pcl::PointXYZRGBNormal>(normals,
```

Comment:
Maybe the function pcl::visualization::PCLVisualizer::Ptr normalsVisualizer should take an input argument which we set to "10 here".

It is a bit hidden this way.

Not strong opinion about this (assuming you will address Tørbjorn's comment on printing out Note: we visualize every 10th normal), but still something to consider.

Reply:
After doing https://github.com/zivid/zivid-cpp-samples/pull/170#discussion_r726849693 and https://github.com/zivid/zivid-cpp-samples/pull/170#discussion_r726913328, I think this looks fine as is now; all visualization code contained in 
```
void visualize(PointCloud<PointXYZRGB>::ConstPtr cloud, PointCloud<PointXYZRGBNormal>::ConstPtr normals)
```
So I suggest we resolve this comment.
Please let me know what you think, whether I should still make `normalsSkipped` an argument to the `visualize()` function.